root/etc/systemd/system/nm-prepare.service

```diff
@@ -2,11 +2,13 @@
 Description=Prepares NetworkManager configuration for primary interface
 Wants=basic.target
 Before=NetworkManager.service
+ConditionKernelCommandLine=BOOTIF
```

Comment:
Until now, we have been failing with exit code 1 when ISO was booted without BOOTIF effectively not configuring the primary interface. This was leaving an error in journal, this way it's prevented completely.